### PR TITLE
fix: specify type for vue-router routes

### PR DESCRIPTION
**Pull request by ferm10n**
Doing so will avoid this cryptic error when using named routes https://github.com/vuejs/router/issues/1213

Hopefully this will help reduce possible sources of friction when trying out a new vuetify project, like I encountered today.

### CODE REVIEW

File: template/typescript/base/src/router/index.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Composables
-import { createRouter, createWebHistory } from 'vue-router'
+import { createRouter, createWebHistory, RouteRecordRaw } from 'vue-router'
 
-const routes = [
+const routes: RouteRecordRaw = [
```
Comment (pascalwill) [inline template/typescript/base/src/router/index.ts:4]: Shouldn't `routes` be declared as array `RouteRecordRaw[]`?

File: template/typescript/custom/router/src/router/index.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Composables
-import { createRouter, createWebHistory } from 'vue-router'
+import { createRouter, createWebHistory, RouteRecordRaw } from 'vue-router'
 
-const routes = [
+const routes: RouteRecordRaw = [
```
Comment (johnleider) [inline template/typescript/custom/router/src/router/index.ts:4]:
```suggestion
const routes: RouteRecordRaw[] = [
```

File: template/typescript/essentials/src/router/index.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // Composables
-import { createRouter, createWebHistory } from 'vue-router'
+import { createRouter, createWebHistory, RouteRecordRaw } from 'vue-router'
 
-const routes = [
+const routes: RouteRecordRaw = [
```
Comment (johnleider) [inline template/typescript/essentials/src/router/index.ts:4]:
```suggestion
const routes: RouteRecordRaw[] = [
```

**Comment by johnleider** [comment]
This PR is no longer necessary, ty for your contribution.

**Pull request closed without merging**